packages/salesforcedx-vscode-core/src/commands/auth/forceAuthAccessTokenLogin.ts

```diff
@@ -38,6 +38,12 @@ export class ForceAuthAccessTokenExecutor extends LibraryCommandletExecutor<
     token?: vscode.CancellationToken
   ): Promise<boolean> {
     const { instanceUrl, accessToken, alias } = response.data;
+    // Do we want to set the alias as default devhub  here? right after login
```

Comment:
Let's break this out into a separate ticket and remove this

Reply:
After review: if this is not commented out, this is causing 1 test failure. Should create and save auth info is calling the save stub twice, and should not be. I was also confused because I'm not seeing `handleAliasAndDefaultSettings` at all on my version of AuthInfo, or in the [code](https://github.com/forcedotcom/sfdx-core/blob/main/src/authInfo.ts).